src/Shared/BuildEnvironmentHelper.cs

```diff
@@ -294,12 +294,22 @@ private static string GetMSBuildExeFromVsRoot(string visualStudioRoot)
             return FileUtilities.CombinePaths(visualStudioRoot, "MSBuild", CurrentToolsVersion, "Bin", "MSBuild.exe");
         }
 
+        private static string GetProcessFromCommandLine()
+        {
+#if FEATURE_GET_COMMANDLINE
+            return Environment.GetCommandLineArgs()[0];
```

Comment:
Does the Mono runtime strip the `mono` prefix from this? .NET Core does not, IIRC.